.gitignore

```diff
@@ -35,7 +35,8 @@ android/.idea
 .env.development.local
 .env.test.local
 .env.production.local
-.vscode
+.vscode/*
+!.vscode/extensions.json
```

Comment:
Ignore all other files inside **.vscode** except for **extensions.json**.
The assumption is that `extensions.json` i.e. recommended extensions can be applicable across developers rather than merely a personal preference.